src/stack.js

```diff
@@ -1,5 +1,28 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+  constructor() {
+    this.dataStore = [];
+    this.top = 0;
```

Comment:
Because I did this word for word from an example from a book that I'm reading and didn't understand what 'this.top' was. I've found out that, that's basically the length of the (in this case) array and since, it's been changed. Had to learn this kind of backwards..